samples/Erlang/sample.app.src

```diff
@@ -0,0 +1,8 @@
+{application, sample,
```

Comment:
Unfortunately, because it's not the same syntax, [it won't be highlighted properly](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fgithub.com%2Ftextmate%2Ferlang.tmbundle%2Fblob%2F39c144ddb7a39bdb967c893a33fda27482beb770%2FSyntaxes%2FHTML%2520%2528Erlang%2529.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fthwg%2Flinguist%2Fblob%2Fapp-src%2Fsamples%2FErlang%2Fsample.app.src&code=). Do you know a Sublime Text/Atom/TextMate package that could be used to highlight those files?